### ISSUE TITLE
toolbox poll snmp broken

### ISSUE THREAD

**Issue by chenshuiluke** [UWICompSociety/infoset#49]
I used the snmp credentials for the calico server in auth.yaml

    ./toolbox.py poll --directory ./infoset/etc
    Traceback (most recent call last):
      File "./toolbox.py", line 3, in <module>
        import yaml
    ImportError: No module named 'yaml'
    [luke:~/git/infoset] i044(+1/-0) 1 ± source venv/bin/activate
    [luke:~/git/infoset] [venv] i044(+1/-0) ± ./toolbox.py poll --directory ./infoset/etc
    ERROR (1007): data_directory: "/opt/infostor/data" in configuration doesn't exist!
    [luke:~/git/infoset] [venv] i044(+1/-0) 3 ± vim
    [luke:~/git/infoset] [venv] i044(+3/-2) ± ./toolbox.py poll --directory ./infoset/etc
    Exception in thread Thread-1:
    Traceback (most recent call last):
      File "/usr/lib64/python3.5/threading.py", line 914, in _bootstrap_inner
        self.run()
      File "/home/luke/git/infoset/infoset/snmp/poll.py", line 46, in run
        snmp_object = snmp_manager.Interact(snmp_params)
      File "/home/luke/git/infoset/infoset/snmp/snmp_manager.py", line 91, in __init__
        if snmp_parameters['snmp_version'] is None:
    TypeError: 'NoneType' object is not subscriptable
    Exception in thread Thread-2:
    Traceback (most recent call last):
      File "/usr/lib64/python3.5/threading.py", line 914, in _bootstrap_inner
        self.run()
      File "/home/luke/git/infoset/infoset/snmp/poll.py", line 46, in run
        snmp_object = snmp_manager.Interact(snmp_params)
      File "/home/luke/git/infoset/infoset/snmp/snmp_manager.py", line 91, in __init__
        if snmp_parameters['snmp_version'] is None:
    TypeError: 'NoneType' object is not subscriptable
    Exception in thread Thread-3:
    Traceback (most recent call last):
      File "/usr/lib64/python3.5/threading.py", line 914, in _bootstrap_inner
        self.run()
      File "/home/luke/git/infoset/infoset/snmp/poll.py", line 46, in run
        snmp_object = snmp_manager.Interact(snmp_params)
      File "/home/luke/git/infoset/infoset/snmp/snmp_manager.py", line 91, in __init__
        if snmp_parameters['snmp_version'] is None:
    TypeError: 'NoneType' object is not subscriptable

**Comment by Pr0x1m4** [comment]
@palisadoes Any ideas on how to fix? Im also getting this error when I try to poll/monitor

**Comment by JA-VON** [comment]
@chenshuiluke @Pr0x1m4  How do you get past 
```
ERROR (1007): data_directory: "/opt/infostor/data" in configuration doesn't exist!
```

**Comment by Pr0x1m4** [comment]
Quick fix is to look in `infoset/infoset/etc/other.yaml`and change
```yaml
data_directory: /opt/infostor/data
web_directory: /opt/infostor/data
```

To
```yaml
data_directory: ./infoset/web
web_directory: ./infoset/web
```

**Comment by JA-VON** [comment]
Ok I thought I had to make the directory in my /opt directory

**Comment by JA-VON** [comment]
here's the output of the command when ran with the changes in the last commit 

<img width="484" alt="screen shot 2016-06-15 at 7 44 45 am" src="https://cloud.githubusercontent.com/assets/6520461/16572646/67670f68-4230-11e6-9adf-bf9915f4e97e.png">

**Comment by palisadoes** [comment]
This should be fixed now after the latest merge. Please retry.

**Comment by Pr0x1m4** [comment]
Closed by #50
